README.md

```diff
@@ -48,6 +48,12 @@ Alternatively, you can use an environment variable:
 GITHUB_TOKEN="<your github token>" gama
 ```
 
+If you have the [GitHub CLI](https://cli.github.com/) installed, you can use
+
+```bash
+export GITHUB_TOKEN="$(gh auth token)" gama
+```
+
```

Comment:
Thank you for your contribution 🍀

I have a few suggestions:

The use of export isn't necessary if the goal is to run GAMA immediately with the environment variable. Therefore, this line should be modified to:

```bash
GITHUB_TOKEN="$(gh auth token)" gama
```
Alternatively, if users prefer exporting the variable separately, they can use:
```bash
export GITHUB_TOKEN="$(gh auth token)"
gama
```

Personally, I think the first approach is more concise and effective for instant execution.

Additionally, as mentioned on line 20, we instruct users to create a token. However, with your method, there's no need to create a new token if they have GitHub CLI installed. Could you please add a note explaining that if users have GitHub CLI, they can use their existing token without creating a new one? This would enhance clarity for anyone setting up the tool.

Thank you again for your valuable contribution! 🛸

Reply:
Dang that was one of the most polite comments I've read in the last week!

Reply:
Just re-read that, yeah, no need for the export if it's not in shell bootstrap a-la .zshrc.
I think somthing like `GITHUB_TOKEN="$(gh auth token)" gama` should be fine.

Reply:
I added some notes, plus I found a bug 😓 it only worked reliably for me when I made a token on the site.

Reply:
│  You have exceeded a secondary rate limit. Please wait a few minutes before you try again. If you reach out to GitHub Support for help, please include the request ID      │
 │  FCFC:236109:869F6B5:1025D3F6:66D7E8B7.]: Repositories cannot be listed

Reply:
Hi, actually it's not a bug. GitHub have rate limits for tokens. You can get this error even if you generate a token. 🍀 
I think this PR ready to merge after remove this line:
`**NOTE: You may still need to generate a token via the GUI if you run into errors**`